### PR TITLE
Better ux for the whole site

### PR DESCRIPTION
**Pull request by pyrech**
- Removed bootstrap
- Used pure.css framework
- Less information to read to understand what is this app for
- More modern design and spaced content
- Fixed some typo
- Used red from the santa logo

Here is the new look and feel :
# Homepage

capture du 2016-10-30 01-07-04

# Form
capture du 2016-10-30 01-13-00

### CODE REVIEW

**Comment by damienalexandre** [comment]
Did you remove the footer with all the legal notices? It was kind of mandatory for the Slack App Directory.

File: views/layout.html.twig [hunk 1]
```diff
@@ -1,178 +1,194 @@
...
+            html, body {
+                background-color: #c4000a;
+            }
+
+            body > .container {
+                background-color: white;
+                position: relative;
+            }
+
+        </style>#}
```
Comment (damienalexandre) [inline views/layout.html.twig]: Remove instead of comment?

File: views/run.html.twig [hunk 2]
```diff
@@ -1,85 +1,92 @@
...
+                    } else {
+                        secretSantaFormSubmitted = true;
+                        document.getElementById('run_btn').disabled = true;
+                        document.getElementById('run_btn').children[0].className = 'fa fa-hourglass-start';
+                    }
+                }, false);
+            </script>
+        {% else %}
+            <div class="error" role="alert">
+                Ooops, we didn't find enough users in your slack team. You need at least 2 users to start a Secret Santa.
```
Comment (damienalexandre) [inline views/run.html.twig]: "slack" takes a capital S.

**Comment by pyrech** [comment]
Nope, just not present in capture :smile: 

Here is how it looks:

capture du 2016-10-30 01-20-19

File: views/run.html.twig (hunk 2 above)
Comment (pyrech) [inline views/run.html.twig]: Fixed two occurrences :+1:

**Comment by tlenclos** [comment]
Good job 👍 🌟

**Comment by tlenclos** [comment]
We could add a photo instead of the big red block. https://unsplash.com/search/christmas

**Comment by tlenclos** [comment]
Or a drawing from Alexandre :trollface: .

**Pull request merged**